### PR TITLE
Change the year of the computation

### LINKED ISSUE #18: Change the year of the computation

### PR DESCRIPTION
**Pull request by ThapeloMasasa**
Fixes #18

### CODE REVIEW

**Comment by PavelMakarchuk** [comment]
Could you show the output if the year 2019 or earlier. We can either deny inputs before 2021 or send a message but I wold rather not have users select pre 2021

File: app.py [hunk 1]
```diff
@@ -107,6 +107,13 @@ def get_net_income(state_code, head_employment_income, spouse_employment_income=
 
 # Create Streamlit inputs for state code, head income, and spouse income.
 state_code = st.text_input("State Code", "CA")
+year_options = ["2023", "2022"]
```
Comment (PavelMakarchuk) [inline app.py:110]: add 2021 and 2024

Comment (MaxGhenis) [reply app.py:110]: we dont need a selector here. 2024 is fine

Comment (ThapeloMasasa) [reply app.py:110]: Made Year Static and removed it from the front end
Screenshot 2024-03-07 at 12 25 07 AM

**Pull request merged**